### DISCUSSION TITLE
Show loading state inside Select component

### DISCUSSION THREAD

**Discussion by ahmadxgani** [mantinedev/mantine#4473] [Q&A]
It would be great if I could show loading when using debounced input on select component. It's possible or is there a workaround for this?

**Comment by ItaiAxelrad** [comment]
Hi, I use `rightSection` with a Loader component. If you need to, you can conditionally supply `data` with a ternary since `disabled` removes the `rightSection` from the Select component.

```js
      <Select
        label='Loading'
        data={loading ? [] : data}
        rightSection={loading && <Loader size='xs' />}
      />
```

**Reply by ahmadxgani** [reply]
Thanks for the answer @ItaiAxelrad 
i will try it!
